Misha/Text/OB-170_fizz_buzz/fizz_buzz_main.py

```diff
@@ -0,0 +1,27 @@
+def fizz_buzz(number):
+    if type(number) != int:
+        raise TypeError('TypeError')
+    else:
+        if number % 3 == 0 and number % 5 != 0:
+            return 'Fizz'
+        elif number % 5 == 0 and number % 3 != 0:
+            return 'Buzz'
+        elif number % 5 == 0 and number % 3 == 0:
+            return 'FizzBuzz'
+        else:
+            return number
```

Comment:
There is no contract in the function. Should return 'Fizz', 'Buzz' or 'FizBuzz'.